src/models.py

```diff
@@ -422,7 +422,7 @@ def build_task_specific_modules(
         These include decoders, linear layers for linear models.
      '''
     task_params = model._get_task_params(task.name)
-    if isinstance(task, SingleClassificationTask):
+    if isinstance(task, SingleClassificationTask) or (isinstance(task, CoLAMinimalPairTask) and task.name == 'cola-pair-tuned'):
```

Comment:
Could you restructure to do the second check within a branch for `isinstance(task, CoLAMinimalPairTask)`? Would like to avoid the check for this branch including very task-specific things (although I know we're guilty of this elsewhere).

Reply:
Sure. (that is so true ; )